source_docs/upgrading/moving-22-30.md

```diff
@@ -0,0 +1,166 @@
+# Moving a site from version 2.2 to the new Version 3.0
+
+This page provides an indepth guide of how to move an existing 2.2.x site up to the latest 3.0 release. Since this
+is a major release you cannot assume that there will be no breaking changes and the complexity of the upgrade will
+vary a lot depending on how much custom code you have in your current site.
+
+You will definitely make the process smoother by taking the following precautions.
+
+1. Make sure you are on the latest stable release of the 2.2.x series
+2. Make sure that you do a full backup of the site database, uploaded files, config files and the extensions directory.
+3. If possible have your current install under git source control and test any changes on a new branch which can then 
+easily be reverted.
+4. If you come across any undocumented issues, let us know by raising an issue on Github
+5. Check that the extensions you need have [versions available for Bolt 3.0][1]
+
+## Getting the latest version via composer
+
+Go to your `composer.json` file in the root of your project and make the following modifications to the `require:` block.
+
+```
+"bolt/bolt": "^3.0@dev",
+"bolt/filesystem": "^2.0@dev",
+"bolt/thumbs": "^3.0@dev",
+```
```

Comment:
@rossriley Just an FYI… the plan is to tag filesystem and thumbs soon, so this won't be needed